src/frontend-samples/transformations-sample/TransformationsClient.ts

```diff
@@ -0,0 +1,56 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+import { AuthorizedClientRequestContext, IncludePrefix, request, Response } from "@bentley/itwin-client";
+import { AuthorizedFrontendRequestContext,IModelApp } from "@bentley/imodeljs-frontend";
+import { AuthorizationClient } from "@itwinjs-sandbox";
+import { GuidString } from "@bentley/bentleyjs-core";
+
+export interface Transformation {
```

Comment:
This dto changed in APIM a bit, check response dto here: https://developer.bentley.com/apis/transformations/operations/get-transformation/